doc/examples/viz_bundles.py

```diff
@@ -0,0 +1,224 @@
+saturation = [0.0, 1.0]  # white to red
+
+lut_cmap = actor.colormap_lookup_table(hue_range=hue,
+                                       saturation_range=saturation)
+
+stream_actor3 = actor.line(bundle_native, fa, linewidth=0.1,
+                           lookup_colormap=lut_cmap)
+bar2 = actor.scalar_bar(lut_cmap)
+
+renderer.add(stream_actor3)
+renderer.add(bar2)
+
+# window.show(renderer, size=(600, 600), reset_camera=False)
+window.record(renderer, out_path='bundle3.png', size=(600, 600))
+
+"""
+.. figure:: bundle3.png
+   :align: center
+
+   **Every point with a color from FA using a non default colomap**.
+
+
+Show every bundle with a specific color
+========================================
+
+You can have a bundle with a specific color. Here orange.
+"""
+
+renderer.clear()
+stream_actor4 = actor.line(bundle_native, (1., 0.5, 0), linewidth=0.1)
+
+renderer.add(stream_actor4)
+
+# window.show(renderer, size=(600, 600), reset_camera=False)
+window.record(renderer, out_path='bundle4.png', size=(600, 600))
+
+"""
+.. figure:: bundle4.png
+   :align: center
+
+   **Entire bundle with a specific color**.
+
+Show every streamline of a bundle with a different color
+========================================================
+
+Let's make a colormap where every streamline of the bundle is colored by its
+length.
+"""
+
+renderer.clear()
+
+from dipy.tracking.streamline import length
+
+lengths = length(bundle_native)
+
+hue = [0.5, 0.5]  # red only
+saturation = [0.0, 1.0]  # black to white
+
+lut_cmap = actor.colormap_lookup_table(
+    scale_range=(lengths.min(), lengths.max()),
+    hue_range=hue,
+    saturation_range=saturation)
+
+stream_actor5 = actor.line(bundle_native, lengths, linewidth=0.1,
+                           lookup_colormap=lut_cmap)
+
+renderer.add(stream_actor5)
+bar3 = actor.scalar_bar(lut_cmap)
+
+renderer.add(bar3)
+
+# window.show(renderer, size=(600, 600), reset_camera=False)
+window.record(renderer, out_path='bundle5.png', size=(600, 600))
+
+"""
+.. figure:: bundle5.png
+   :align: center
+   **Color every streamline by the length of the streamline **
+
+
+Show every point of every streamline with a different color
+============================================================
+
+In this case in which we want to have a color per point and per streamline,
+we can create a list of the colors to correspond to the list of streamlines
+(bundles). Here in ``colors`` we will insert some random RGB colors.
+"""
+
+renderer.clear()
+
+colors = [np.random.rand(*streamline.shape) for streamline in bundle_native]
+
+stream_actor6 = actor.line(bundle_native, colors, linewidth=0.2)
+
+renderer.add(stream_actor6)
+
+# window.show(renderer, size=(600, 600), reset_camera=False)
+window.record(renderer, out_path='bundle6.png', size=(600, 600))
+
+"""
+.. figure:: bundle6.png
+   :align: center
+
+   **Random colors per points per streamline**.
+
+In summary, we showed that there are many useful ways for visualizing maps
+on bundles.
+
```

Comment:
Yes. This colorbar is just an actor so you can position it where you want. Also in the next PR we will have a container object that allows you to grid things together. Finally, you can also create a colorbar widget in vtk which you are then able to drag interactively. I think the best option will be the container object because it will align the colorbar and the actors correctly. We should add an example in the container tutorial @MarcCote :)